.github/workflows/ci.yml

```diff
@@ -4,42 +4,84 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    branches: [main, dev/foundry]
```

Comment:
@DhairyaSethi can you create another PR that changes this to dev once merged it, so fork main -> dev, then push the branch with branch as dev.

Reply:
#64